addons/website_sale/static/src/js/website_sale_tour.js

```diff
@@ -0,0 +1,395 @@
+        content: "Add one more Storage Box",
+        trigger: '.js_add_cart_json:eq(1)',
+    },
+    {
+        content: "Check b2c Tax-Included Prices",
+        trigger: '.product_price .oe_price .oe_currency_value:containsExact(90.85)',
+    },
+    {
+        content: "Click on add to cart",
+        trigger: '#add_to_cart',
+    },
+    {
+        content: "Check for 2 products in cart and proceed to checkout",
+        extra_trigger: '#cart_products tr:contains("Storage Box") input.js_quantity:propValue(2)',
+        trigger: 'a[href*="/shop/checkout"]',
+    },
+    {
+        content: "Check Price b2c total",
+        trigger: 'tr#order_total_untaxed .oe_currency_value:containsExact(158.00)',
+    },
+    {
+        content: "Check Price b2c Sale Tax(15%)",
+        trigger: 'tr#order_total_taxes .oe_currency_value:containsExact(23.70)',
+    },
+    {
+        content: "Check Price b2c Total amount",
+        trigger: 'tr#order_total .oe_currency_value:containsExact(181.70)',
+    },
+    {
+        content: "Click on Login Button",
+        trigger: '.oe_cart a.btn:contains("Log In")',
+    },
+    {
+        content: "Submit login",
+        trigger: '.oe_login_form',
+        run: function (){
+            $('.oe_login_form input[name="login"]').val("abc@odoo.com");
+            $('.oe_login_form input[name="password"]').val("1admin@admin");
+            $('.oe_login_form').submit();
+        },
+    },
+    {
+        content: "Add new shipping address",
+        trigger: '.one_kanban form[action^="/shop/address"] .btn',
+    },
+    {
+        content: "Fulfill shipping address form",
+        trigger: 'select[name="country_id"]',
+        run : function () {
+            $('input[name="name"]').val('ghi');
+            $('input[name="phone"]').val('7777777777');
+            $('input[name="street"]').val('2,wxy');
+            $('input[name="city"]').val('xyz');
+            $('#country_id option:eq(1)').attr('selected', true);
+        },
+    },
+    {
+        content: "Click on next button",
+        trigger: '.oe_cart .btn:contains("Next")',
+    },
+    {
+        content: "Select `Wire Transfer` payment method",
+        trigger: '#payment_method label:contains("Wire Transfer")',
+    },
+    {
+        content: "Pay Now",
+        extra_trigger: '#payment_method label:contains("Wire Transfer") input:checked,#payment_method:not(:has("input:radio:visible"))',
+        trigger: 'button[id="o_payment_form_pay"]:visible:not(:disabled)',
+    },
+    {
+        content: "Open Dropdown for See quotation",
+        extra_trigger: '.oe_cart .oe_website_sale_tx_status',
+        trigger: '#top_menu li.dropdown:visible a:contains("abc")',
+    },
+    {
+        content: "My account",
+        extra_trigger: '#top_menu li.dropdown .js_usermenu.show',
+        trigger: '#top_menu .dropdown-menu a[href="/my/home"]:visible',
+    },
+    {
+        content: "See Quotations",
+        trigger: '.o_portal_docs a:contains("Quotations")',
```

Comment:
`.o_portal_docs a:contains("Quotations") .badge:containsExact(2)` if the goal is to check the quotation was correctly created (same step works fine the first time as if exists it means there is onle quotation at least but in this case this is not enough to ensure the second one was created)